app/lib/pre_assembly/digital_object.rb

```diff
@@ -290,7 +290,7 @@ def assembly_workflow_url
 
     def retry_handler(method_name, logger, params = {})
       Proc.new do |exception, attempt_number, total_delay|
-        log("      ** #{method_name} FAILED **; with params of #{params.inspect}; and trying attempt #{attempt_number} of #{Dor::Config.dor.num_attempts}; delayed #{Dor::Config.dor.total_delay} seconds")
+        log("      ** #{method_name} FAILED **; with params of #{params.inspect}; and trying attempt #{attempt_number} of #{Dor::Config.dor_services.num_attempts}; delayed #{total_delay} seconds")
```

Comment:
caught this nifty "total_delay" goof too.  But I agree with @atz that this retry_handler isn't doing much anyway.

Reply:
I think you misread what the Proc is doing.  The signature is set by the `retries` gem and the `total_delay` is one of the parameters passed in on line 292.  The total delay would change between retries 1, 2 and 3.  The `Dor::Config.dor.total_delay` does not.  I think the `dor` => `dor_services` shift is good, but the old `total_delay` is what we want.

Reply:
How/Where is Dor::Config.dor.total_delay defined?   Is there a Dor::Config.dor  distinct from DC.dor_services?   I am confused.

Reply:
No, wait, I read the diff backwards.  I'm the confused one, sorry.  👍